src/components/LobbyErrorDialog.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <BaseDialog
+    id="ErrorDialog"
+    :model-value="showLobbyError"
+    title="Oops!"
+    :opacity="1"
+    data-cy="create-game-dialog"
+  >
+    <template #body>
+      <div>
+        <h4>Cannot Join Lobby {{ errorMessage }}</h4>
+      </div>
+    </template>
+    <template #actions>
+      <v-btn
+        data-cy="how-it-works-okay"
+        color="surface-1"
+        variant="flat"
+        class="text-surface-2"
+        @click="close"
+      >
+        Go Home
+      </v-btn>
+    </template>
+  </BaseDialog>
+</template>
+
+<script setup>
+import BaseDialog from '@/components/BaseDialog.vue';
+import { useRouter, useRoute } from 'vue-router';
+const { showLobbyError } = defineProps({ showLobbyError: {type: Boolean, default: false } });
```

Comment:
This is interesting. I believe this is the deprecated [reactive props destructuring](https://vuejs.org/guide/extras/reactivity-transform#reactive-props-destructure) feature of vue, correct? This has been removed from vue as of 3.4 so I think we should replace it so that we refrain from destructuring. It looks like we don't need the props in js atm (just the template) so I think we'd want this
```suggestion
defineProps<{
  showLobbyError: {
    type: Boolean,
    default: false
  }
}>();
```